apps/ejabberd/src/mod_mam_muc.erl

```diff
@@ -331,7 +337,12 @@ is_user_identity_hidden(From, ArcJID) ->
 
 -spec can_access_room(From :: ejabberd:jid(), To :: ejabberd:jid()) -> boolean().
 can_access_room(From, To) ->
-    ejabberd_hooks:run_fold(can_access_room, To#jid.lserver, false, [From, To]).
+    Stanza = mongoose_stanza:new(), % some day original stanza will reach this stage
```

Comment:
We do, because hook handlers have been changed to accept a 'mongoose_stanza' as an accumulator. Since we don't have original stanza here we have to make one. After stage II original stanza will travel everywhere and all those wrappers around hook calls will go away. I've done the same in dozens of places, only here I put a comment.